modules/core/perf/func_tests/test_task.hpp

```diff
@@ -27,6 +31,9 @@ class TestTask : public ppc::core::Task {
     for (unsigned i = 0; i < taskData->inputs_count[0]; i++) {
       output_[0] += input_[i];
     }
+    if (enableSleep) {
```

Comment:
I'd suggest to inherit dummy test task that performs sleep from TestTask class and use it where it is needed

Reply:
@aobolensk corrected